cookiecutter/log.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+"""Module for up logging set up."""
```

Comment:
The grammar isn't correct. What are you trying to say?

``Module for setting up logging.``